pkg/scannerv4/mappers/mappers.go

```diff
@@ -347,6 +348,11 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 				Str("severity", v.Severity).
 				Msg("missing severity and/or CVSS score(s): proceeding with partial values")
 		}
+		var preferredCVSS *v4.VulnerabilityReport_Vulnerability_CVSS
+		if len(metrics) > 0 {
+			// The preferred CVSS metrics will always be stored at the first index.
```

Comment:
Can we have this comment in Line 674 also? In case people change logics in `cvssMetrics()` so they would know.

Reply:
I have it in the godoc for `cvssMetrics()`

```
// When not empty, the first entry is the "preferred" metric.
```

Is that ok?